tc_solaris.go

```diff
@@ -0,0 +1,75 @@
+package console
+
+import (
+	"os"
+
+	"golang.org/x/sys/unix"
+)
+
+func tcget(fd uintptr, p *unix.Termios) error {
+	var err error
+	p, err = unix.IoctlGetTermios(int(fd), unix.TCGETS)
+	return err
+}
+
+func tcset(fd uintptr, p *unix.Termios) error {
+	return unix.IoctlSetTermios(int(fd), unix.TCSETS, p)
+}
+
+func tcgwinsz(fd uintptr) (WinSize, error) {
+	var ws WinSize
+
+	uws, err := unix.IoctlGetWinsize(int(fd), unix.TIOCGWINSZ)
```

Comment:
ah.  so basically update the vendored version of golang/x/sys in containerd before vendoring in this change.  i can do that.  (i was thinking of golang/x/sys as tied to a go release because in our development solaris builds we deliver it along with go itself because the official upstream versions don't yet have SPARC support.)